test/unit/http/ldp/PostOperationHandler.test.ts

```diff
@@ -21,7 +20,10 @@ describe('A PostOperationHandler', (): void => {
     body = new BasicRepresentation('', 'text/turtle');
     operation = { method: 'POST', target: { path: 'http://test.com/foo' }, body, conditions, preferences: {}};
     store = {
-      addResource: jest.fn(async(): Promise<ResourceIdentifier> => ({ path: 'newPath' } as ResourceIdentifier)),
+      addResource: jest.fn(async(): Promise<Record<string, RepresentationMetadata>> => ({
+        newPath: new RepresentationMetadata({ path: 'newPath' }).add(SOLID_AS.terms.Activity, AS.Create),
+        parent: new RepresentationMetadata({ path: 'parent' }).add(SOLID_AS.terms.Activity, AS.Update),
```

Comment:
would use actual URLs for the identifiers to prevent confusion when reading later.